# Unique session id
int32 session_id

# Contains auxiliary diagnostic information (in json format)
string params

---

# Task specific observation (in json format)
string observation

---
